src/components/eventing/TriggerForm.jsx

```diff
@@ -25,13 +25,13 @@ const TriggerForm = ({ handleSubmit, eventTypes, initialEventType, internalToken
   const [data, setData] = useState("{}")
   const [eventResponse, setEventResponse] = useState(null)
   const [triggeredEventOnce, setTriggeredEventOnce] = useState(false)
-  const useInternalToken = useSelector(state => get(state, "uiState.eventing.useInternalToken", true))
-  const token = useSelector(state => get(state, "uiState.eventing.token", ""))
+  const useInternalToken = getLastUsedValues(projectId).useInternalToken
```

Comment:
This value should be first coming off from redux. I think the best place to set this would be the `store.js` file